src/nvim/fold.c

```diff
@@ -1880,6 +1880,7 @@ void foldtext_cleanup(char_u *str)
 /*
  * Update the folding for window "wp", at least from lines "top" to "bot".
  * Return TRUE if any folds did change.
+ * Indent Expr Marker Syntax (IEMS)
```

Comment:
this line was confusing until I noticed the function name. This might be clearer:

```suggestion
 * IEMS = "Indent Expr Marker Syntax"
```

Should change the comment to `//` or `///` form also.